fastapi/dependencies/utils.py

```diff
@@ -776,6 +781,23 @@ def get_body_field(*, dependant: Dependant, name: str) -> Optional[ModelField]:
         ]
         if len(set(body_param_media_types)) == 1:
             BodyFieldInfo_kwargs["media_type"] = body_param_media_types[0]
+    if is_form_data(BodyFieldInfo):
+        try:
+            from multipart import QuerystringParser  # check if there's an import
+
+            QuerystringParser(  # check if correct import using python-multipart function
+                {}
+            )
+        except ModuleNotFoundError:
+            error = "Form data requires [python-multipart] to be installed."
+            logger.error(error)
+            raise RuntimeError(error)
+        except ImportError:
+            error = """Form data requires [python-multipart] to be installed. Currently 
```

Comment:
If you print `error` you'll see that there are a lot of spaces on it.
```suggestion:-0+2
            error = (
                "Form data requires [python-multipart] to be installed. Currently "
                "[multipart] is installed and not compatible with [python-multipart]. "
                "Uninstall [multipart] and then install [python-multipart]."
            )
```

Reply:
I didn't know you could do this to get rid of the spaces; thanks for this suggestion 😄

Reply:
I don't know how to create a multi-line suggestion, does anyone know?

Reply:
Oh I didn't realize the rest of the lines were there too; I'll fix it and make another push, sorry about that